NOCList/NOCList/LSIAgent.m

```diff
@@ -0,0 +1,97 @@
+//
+//  LSIAgent.m
+//  NOCList
+//
+//  Created by Shawn Gee on 5/11/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+#import "LSIAgent.h"
+
+@implementation LSIAgent
+
+-(instancetype)initAgentWithCoverName:(NSString *)coverName
+                             realName:(NSString *)realName
+                          accessLevel:(int)accessLevel
+                        isCompromised:(BOOL)isCompromised {
+    self = [super init];
+    if (self != nil) {
+        _coverName = coverName;
+        _realName = realName;
+        _accessLevel = [NSNumber numberWithInt:accessLevel];
+        _isCompromised = [NSNumber numberWithBool:isCompromised];
+    }
+    return self;
+}
+
++(int)numberOfCompromisedAgentsInArray:(NSArray *)agents {
+    int compromisedAgentCount = 0;
+    
+    for (LSIAgent *agent in agents) {
+        if (agent.isCompromised.boolValue == YES) {
+            compromisedAgentCount++;
+        }
+    }
+    
+    return compromisedAgentCount;
+}
+
++(int)numberOfCleanAgentsInArray:(NSArray *)agents {
+    int cleanAgentCount = 0;
+    
+    for (int i = 0; i<agents.count; i++) {
```

Comment:
Generally I've seen a space put around comparison operators (eg `i < agents.count`); not a huge deal of course, but as Paul often says, consistency is key.